src/main/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGAViaProcessBuilderOnSpark.java

```diff
@@ -75,20 +75,52 @@
-    @VisibleForTesting static final boolean TURN_ON_TRANSITIVE_REDUCTION = false;
+    //----------------------------------------------------------------------------------------------------------------//
+    // Result-affecting parameters for use in various SGA modules with default values.
+    // Long names are the same as those in SGA documentation, except using camel case in place of dashes.
+    //----------------------------------------------------------------------------------------------------------------//
+
+    @Argument(doc       = "Minimum overlap parameter value for steps: fm-merge, overlap, and assemble.",
+              shortName = "mol",
+              fullName  = "minOverlap",
+              optional  = true)
+    public static final int MIN_OVERLAP_IN_FILTER_OVERLAP_ASSEMBLE = 55;
+    @Argument(doc       = "Require at least this number of coverage for each kmer in a read in order to pass filter.",
+              shortName = "kfreq",
+              fullName  = "kmerThreshold",
+              optional  = true)
+    public static final int FILTER_STEP_KMER_FREQUENCY_THRESHOLD = 1;
+    @Argument(doc       = "The maximum error rate allowed to consider two sequences aligned in the overlap step.",
+              shortName = "err",
+              fullName  = "errorRate",
+              optional  = true)
+    public static final double OVERLAP_STEP_ERROR_RATE = 0.02;
+    @Argument(doc       = "Remove variation only if the divergence between sequences when only counting indels is less than this value.",
+              shortName = "maxGapDiv",
+              fullName  = "maxGapDivergence",
+              optional  = true)
+    public static final double ASSEMBLE_STEP_MAX_GAP_DIVERGENCE = 0.0;
+    @Argument(doc       = "Remove terminal branches only if they are less than this number of bases long. Ineffective when terminal branch removal is turned off.",
+              shortName = "minTailLen",
+              fullName  = "minBranchLength",
+              optional  = true)
+    public static final int ASSEMBLE_STEP_MIN_BRANCH_TAIL_LENGTH = 50;
+    @Argument(doc       = "Cut off terminal branches in this number of rounds. When set to zero, removal is turned off.",
+              shortName = "cutTail",
+              fullName  = "cutTerminal",
+              optional  = true)
+    public static final int CUT_OFF_TERMINAL_BRANCHES_IN_N_ROUNDS = 0;
+    @Argument(doc       = "Remove transitive edges from the graph.",
+              shortName = "rmTransEdges",
+              fullName  = "transitiveReduction",
+              optional  = true)
+    public static final boolean TURN_ON_TRANSITIVE_REDUCTION = false;
 
 
     // for developer performance debugging use
     private static final DateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
     private static final Logger logger = LogManager.getLogger(RunSGAViaProcessBuilderOnSpark.class);
+    private static final int files_per_partition = 5; // performance tuning
```

Comment:
I think in our usual style we make static final constants all caps: FILES_PER_PARTITION